docs/control-structures.rst

```diff
@@ -385,6 +385,9 @@ There are two other ways to trigger exceptions: The ``revert`` function can be u
 revert the current call. It is possible to provide a string message containing details about the error
 that will be passed back to the caller.
 
+.. warning::
```

Comment:
Please undo this move, it refers to the paragraph after the note.

Reply:
Looks like this was fixed by #6846 as in develop it's in the right place now.